environment-approvals.yml

```diff
@@ -0,0 +1,3 @@
+prod:
+  - https://github.com/HMCTS/cnp-plum-recipes-service.git
+  - https://github.com/HMCTS/cnp-plum-shared-infrastructure.git
```

Comment:
I also wonder whether we should have a yaml object to make it more extensible, 
```suggestion
  - repo: https://github.com/HMCTS/cnp-plum-shared-infrastructure.git
```

Reply:
was thinking similar - but was wondering if it's too much thinking ahead